framework/source/class/qx/event/message/Bus.js

```diff
@@ -31,83 +36,76 @@ qx.Class.define("qx.event.message.Bus",
   {
 
     /**
-     * gets the hash map of message subscriptions
-     *
-     * @return {Map} with registered subscriptions. The key is the
-     *    <code>message</code> and the value is a map with <code>{subscriber: {Function},
-     *    context: {Object|null}}</code>.
+     * Shorthand method for {@link qx.event.message.Bus.getSubscription}
+     * @return {Object}
      */
     getSubscriptions : function() {
       return this.getInstance().getSubscriptions();
     },
 
-
     /**
-     * subscribes to a message
-     *
-     * @param message {String} name of message, can be truncated by *
-     * @param subscriber {Function} subscribing callback function
+     * Shorthand method for {@link qx.event.message.Bus.subscribe}
+     * @param topic {String|RegExp}
+     * @param subscriber {Function} Message handler function
      * @param context {Object} The execution context of the callback (i.e. "this")
      * @return {Boolean} Success
      */
-    subscribe : function(message, subscriber, context)
+    subscribe : function(topic, subscriber, context)
     {
-      return this.getInstance().subscribe(message, subscriber, context);
+      return this.getInstance().subscribe.apply(this.getInstance(), arguments);
+    },
 
+    /**
+     * Shorthand method for {@link qx.event.message.Bus.subscribeOnce}
+     * @param topic {String|RegExp}
+     * @param subscriber {Function} Message handler function
+     * @param context {Object} The execution context of the callback (i.e. "this")
+     * @return {Boolean} Success
+     */
+    subscribeOnce : function(topic, subscriber, context)
+    {
+      return this.getInstance().subscribeOnce.apply(this.getInstance(), arguments);
     },
 
     /**
-     * checks if subscription is already present
-     * if you supply the callback function, match only the exact message monitor
-     * otherwise match all monitors that have the given message
-     *
-     * @param message {String} Name of message, can be truncated by *
-     * @param subscriber {Function} Callback Function
-     * @param context {Object} execution context
-     * @return {Boolean} Whether monitor is present or not
+     * Shorthand method for {@link qx.event.message.Bus.checkSubscription}
+     * @param topic {String|RegExp} The topic that has been used when subscribing
```

Comment:
The parameter is documented with name `topic` while it's still called `message` in the function parameter list below.

Reply:
Updated the method signature